src/test/java/uk/gov/ons/ctp/response/action/service/ActionProcessingServiceTest.java

```diff
@@ -217,170 +304,35 @@ public void testProcessActionRequestActionPersistingActionThrowsException() thro
-        .createNewCaseEvent(any(Action.class), eq(CategoryDTO.CategoryName.ACTION_CREATED));
-  }
-
-  /**
-   * Happy path for an action linked to a case for a PARENT sample unit (a H one), ie we go all the
-   * way to producing an ActionRequest and publishing it.
-   */
-  @Test
-  public void testProcessActionRequestHappyPathParentUnit() throws CTPException {
-    // Start of section to mock responses
-    ActionRequestContext context = new ActionRequestContext();
-
-    final ActionPlan actionPlan =
-        ActionPlan.builder().name(ACTION_PLAN_NAME).id(UUID.randomUUID()).build();
-    context.setActionPlan(actionPlan);
-    context.setCaseDetails(caseDetailsDTOs.get(0));
-    context.setParentParty(partyDTOs.get(0));
-    context.setCollectionExercise(collectionExerciseDTOs.get(0));
-    context.setSurvey(surveyDTOs.get(0));
-    context.setSampleUnitType(SampleUnitType.B);
-
-    when(actionSvcStateTransitionManager.transition(
-            any(ActionDTO.ActionState.class), any(ActionDTO.ActionEvent.class)))
-        .thenReturn(ActionDTO.ActionState.PENDING);
-    when(validator.validate(any(ActionType.class), any(ActionRequest.class))).thenReturn(true);
-    // End of section to mock responses
-
-    // Start of section to run the test
-    final Action action = new Action();
-    action.setId(ACTION_ID);
-    action.setActionType(
-        ActionType.builder().responseRequired(Boolean.TRUE).handler(ACTIONEXPORTER).build());
-    action.setActionPlanFK(ACTION_PLAN_FK);
-    action.setCaseId(CASE_ID);
-    action.setPriority(1);
-    context.setAction(action);
-
     when(this.decoratorContextFactory.getActionRequestDecoratorContext(any(Action.class)))
         .thenReturn(context);
 
-    businessActionProcessingService.processActionRequest(action);
-    // End of section to run the test
+    // When
+    businessActionProcessingService.processActionRequest(contextAction);
 
-    // Start of section to verify calls
+    // Then
     verify(actionSvcStateTransitionManager, times(1))
         .transition(
             any(ActionDTO.ActionState.class), eq(ActionDTO.ActionEvent.REQUEST_DISTRIBUTED));
     verify(actionRepo, times(1)).saveAndFlush(any(Action.class));
-    verify(caseSvcClientService, times(1))
-        .createNewCaseEvent(any(Action.class), eq(CategoryDTO.CategoryName.ACTION_CREATED));
-    // TODO Be more specific on the Action below once CTPA-1390 has been discussed & implemented
+    verify(partySvcClientService, never()).getParty(eq(SAMPLE_UNIT_TYPE_HI), any(UUID.class));
     verify(actionInstructionPublisher, times(1))
         .sendActionInstruction(
-            eq(ACTIONEXPORTER),
+            any(String.class),
             any(uk.gov.ons.ctp.response.action.message.instruction.Action.class));
-  }
-
-  /**
-   * Happy path for an action linked to a case for a CHILD sample unit (a HI one), ie we go all the
-   * way to producing an ActionRequest and publishing it.
-   */
-  @Test
```

Comment:
What's "HI" - is that social individual or something? Do we need this test?

Reply:
Talked to Dave about this from capabilities. HI cases aren't in use yet and we're good to remove the depreciated functionality for it